src/SonarScanner.MSBuild.PreProcessor/WebServer/SonarWebServer.cs

```diff
@@ -117,14 +114,11 @@ public async Task<IList<SonarRule>> GetRules(string qProfile)
 
         public async Task<IEnumerable<string>> GetAllLanguages()
         {
-            var uri = GetUri("api/languages/list");
-            return await ExecuteWithLogs(async () =>
-            {
-                var contents = await downloader.Download(uri);
+            var uri = "api/languages/list";
```

Comment:
this can be inlined. It's used only once and I think the code would be easier to read